# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/window.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/window.h
--- a/security/sandbox/chromium/sandbox/win/src/window.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/window.h	Wed Jun 12 11:10:48 2019 +0100
@@ -6,6 +6,7 @@
 #define SANDBOX_SRC_WINDOW_H_
 
 #include <windows.h>
+
 #include <string>
 
 #include "base/strings/string16.h"
@@ -13,27 +14,27 @@
 
 namespace sandbox {
 
-  // Creates a window station. The name is generated by the OS. The security
-  // descriptor is based on the security descriptor of the current window
-  // station.
-  ResultCode CreateAltWindowStation(HWINSTA* winsta);
+// Creates a window station. The name is generated by the OS. The security
+// descriptor is based on the security descriptor of the current window
+// station.
+ResultCode CreateAltWindowStation(HWINSTA* winsta);
 
-  // Creates a desktop. The name is a static string followed by the pid of the
-  // current process. The security descriptor on the new desktop is based on the
-  // security descriptor of the desktop associated with the current thread.
-  // If a winsta is specified, the function will switch to it before creating
-  // the desktop. If the functions fails the switch back to the current winsta,
-  // the function will return SBOX_ERROR_FAILED_TO_SWITCH_BACK_WINSTATION.
-  ResultCode CreateAltDesktop(HWINSTA winsta, HDESK* desktop);
+// Creates a desktop. The name is a static string followed by the pid of the
+// current process. The security descriptor on the new desktop is based on the
+// security descriptor of the desktop associated with the current thread.
+// If a winsta is specified, the function will switch to it before creating
+// the desktop. If the functions fails the switch back to the current winsta,
+// the function will return SBOX_ERROR_FAILED_TO_SWITCH_BACK_WINSTATION.
+ResultCode CreateAltDesktop(HWINSTA winsta, HDESK* desktop);
 
-  // Returns the name of a desktop or a window station.
-  base::string16 GetWindowObjectName(HANDLE handle);
+// Returns the name of a desktop or a window station.
+base::string16 GetWindowObjectName(HANDLE handle);
 
-  // Returns the name of the desktop referenced by |desktop|. If a window
-  // station is specified, the name is prepended with the window station name,
-  // followed by a backslash. This name can be used as the lpDesktop parameter
-  // to CreateProcess.
-  base::string16 GetFullDesktopName(HWINSTA winsta, HDESK desktop);
+// Returns the name of the desktop referenced by |desktop|. If a window
+// station is specified, the name is prepended with the window station name,
+// followed by a backslash. This name can be used as the lpDesktop parameter
+// to CreateProcess.
+base::string16 GetFullDesktopName(HWINSTA winsta, HDESK desktop);
 
 }  // namespace sandbox
 